entry.go

```diff
@@ -30,7 +30,7 @@ func (e Entry) Alias() string {
 //	The call is ACID, regardless of the type of the entry and a transaction will be created if need be
 func (e Entry) Remove() error {
 	err := C.qdb_remove(e.handle, C.CString(e.alias))
-	return makeErrorOrNil(err)
+	return makeErrorOrNil(ErrorType(err))
```

Comment:
I don't understand why you changed it. Couldn't it be left as it was and add a new function that takes `ErrorType` if really necessary?